README.md

```diff
@@ -109,6 +109,15 @@ default values only after having loaded this script into your ZSH session.
   causes this script to perform a case-insensitive search. See the "Globbing
   Flags" section in the zshexpn(1) man page to learn about the kinds of
   values you may assign to this variable.
+  
+* `HISTORY_SUBSTRING_SEARCH_METHOD` is a global variable that defines
+  how the command history will be searched for your query.
+  * 'default' - find history by string
+    * e.g. 'a b c' will match 'xxxA B Cxxx'
+  * 'fuzzy_words' - find history by fuzzy word matching
+    * e.g. 'ab c' will match 'xxxABxxxCxxx'
+  * 'fuzzy_chars' - find history by fuzzy character matching
+    * e.g. 'abc' will match 'xxxAxxxBxxxCxxx'
 
```

Comment:
Sounds good, but remember we still need to handle the "whitespace in pattern" case properly.